graphqlapi/src/main/java/io/stargate/graphql/persistence/schemafirst/SchemaSourceDao.java

```diff
@@ -116,7 +136,7 @@ BoundQuery schemaQueryWithSpecificVersion(String namespace, UUID uuid) {
         .select()
         .column(VERSION_COLUMN_NAME, CONTENTS_COLUMN_NAME)
         .from(namespace, TABLE_NAME)
-        .where(KEY_CONDITION)
+        .where(KEY_COLUMN_NAME, Predicate.EQ, UNIQUE_KEY)
```

Comment:
Are you removing the static for consistency?

Reply:
Yes. I made you inline the other one on your last PR, so it didn't make sense to keep this one either.